dev_guide/secrets.adoc

```diff
@@ -80,6 +108,30 @@ $ oc create -f <filename>
 ----
 ====
 
+[[types-of-secrets]]
+=== Types of Secrets
+
+The value in the `type` field is a hint at the structure of the secret's key names and values.
+
+Some types trigger minimal server-side validation to ensure the presence of specific key names in the secret data:
+
+* `kubernetes.io/service-account-token`. Uses a xref:../admin_guide/service_accounts.adoc#enabling-service-account-authentication[service account token].
+* `kubernetes.io/dockercfg`. Uses the xref:../dev_guide/managing_images.adoc#allowing-pods-to-reference-images-from-other-secured-registries[*_.dockercfg_* file] 
+for required Docker credentials.
+* `kubernetes.io/dockerconfigjson`. Uses the xref:../dev_guide/builds/build_inputs.adoc#using-docker-credentials-for-private-registries[*_.docker/config.json_* file] 
+for required Docker credentials. 
+* `kubernetes.io/basic-auth`. Use with xref:../dev_guide/builds/build_inputs.adoc#source-secrets-basic-authentication[Basic Authentication].
+* `kubernetes.io/ssh-auth`. Use with xref:../dev_guide/builds/build_inputs.adoc#source-secrets-ssh-key-authentication[SSH Key Authentication].
+* `kubernetes.io/tls`. Use with xref:../dev_guide/builds/build_inputs.adoc#source-secrets-trusted-certificate-authorities[TLS certificate authorities]
+
+You can specify other arbitrary types (such as `example.com/my-secret-type`), which are not enforced server-side, 
```

Comment:
", which are not enforced server-side, " seems vital to the meaning of the sentence. If you agree, s/which/that, and fix the commas.
Also, IBM Style frowns on parentheses.